### ISSUE TITLE
is it possible to show the ancestors of two at the same time?

### ISSUE THREAD

**Issue by 1E61** [BALKANGraph/FamilyTreeJS#25]
Hi, 
is it possible to show the ancestors of two at the same time?

![image](https://user-images.githubusercontent.com/13537222/181587274-a2715aa8-0c38-4186-882d-01f56a2b14b0.png)

**Comment by ZornitsaPesheva** [comment]
No, it is not possible .Only the parents on the one of the partners can be expanded at a time.

**Comment by refton** [comment]
@ZornitsaPesheva, Hi!
Did I understand correctly that there is no possibility to show all tree expanded?
If it's so, is it in your future plans?
I think it looks like necessary option for family tree))

**Comment by ZornitsaPesheva** [comment]
We may implement this, but we don't have such plan for the near future.

**Comment by refton** [comment]
@ZornitsaPesheva, thanks)
As for me - it will be a very useful feature, because you want see how tree grow after each node adding.
I will wait for the update with this option

**Comment by daxxac** [comment]
> We may implement this

I'd love to have that feature too.

**Comment by ZornitsaPesheva** [comment]
Yes, we already are working on that, but it will not be released soon, as it is actually totally new implementation and will take time.

**Comment by dimger** [comment]
Do we have any news on this one?

**Comment by ZornitsaPesheva** [comment]
Still working. You can follow us on [Facebook](https://www.facebook.com/BALKAN.App.Inc) and [LinkedIn](https://www.linkedin.com/company/balkanapp) for any news.
